packages/next/src/server/import-overrides.ts

```diff
@@ -97,23 +103,3 @@ function overrideReact() {
   }
 }
 overrideReact()
-
-mod._resolveFilename = function (
```

Comment:
Node.js `--experimental-loader` does not apply to CommonJS as far as I could find, so this one is still needed too afaik.

In Node.js 20.6.0 support for loaders with commonjs was added.

Reply:
This is moved to the require-hook